libioc/Firewall.py

```diff
@@ -82,6 +82,16 @@ def ensure_firewall_enabled(self) -> None:
             logger=self.logger
         )
 
+    def rules(
+        self
+    ) -> str:
+        """Output lists of firewall rule."""
+        command = [
+            self.IPFW_COMMAND,
+            "-a", "list"
```

Comment:
This command generates a generic list of firewall rules on a host, while the Firewall module should focus on rules that are managed by libioc. Those rules begin at a certain offset - finding the upper boundary is a tougher question.

Reply:
right.
Honestly, I think shouldn't setting firewall for host via libioc too.
But, I think correct that setting(or confirm) firewall related host and container on same library. Because It is layer, If library doesn't deal both, that is irrelevant.